onnxscript/rewriter/pattern.py

```diff
@@ -634,6 +634,20 @@ def _is_pattern_variable(x: Any) -> bool:
     return type(x) is ValuePattern
 
 
+class AnyValue(ValuePattern):
+    """Represents a pattern that matches against any value."""
+
+    def __init__(self) -> None:
+        super().__init__(None)
+
+    def clone(self, node_map: dict[NodePattern, NodePattern]) -> AnyValue:
+        # A single instance of AnyValue suffices.
+        return self
+
+
+any_value = AnyValue()
```

Comment:
Any suggestions for alternative names? Another possibility is `ignore`. Or, would altering the casing to `ANY_VALUE` be better?

Reply:
I like `ANY_VALUE` or more simply `ANY`

Reply:
Leaning towards 'ANY_VALUE' as well

Reply:
done